### ISSUE TITLE
[🐞]: Cage with allow_all_mobs set to true unable to capture some mobs

### ISSUE THREAD

**Issue by thekrugers** [MehVahdJukaar/Supplementaries#1075]
### Before Continuing:

- [X] Check you are using the latest version of the mods and its dependencies
- [X] Always include a latest.log if you are crashing
- [X] Remove mod that enhances Minecraft: Optifine, Sodium, others. The issue persists.
- [X] If you are unsure which mod is the culprit.
Disable all of your mods and enable them 1-2 mods each time to isolate the culprit
- [X] Confirm that there is no existing issue with a similar description submitted in the list of issues.

### Version

1.20.1-FABRIC

### Supplementaries Version

supplementaries-1.20-2.8.15

### Moonlight Lib Version

moonlight-1.20-2.11.41

### Issue Detail

I have "allow_all_mobs": true in my supplementaries-common.json, but I get the "Can only capture small animals" message when trying to capture some mobs. I've seen this message with vanilla parrots and the robin from the Wildlife mod.
In the same world before updating, where I ran supplementaries-1.20-2.7.32, I was able to capture these mobs.
If it helps, this is on a multiplayer server, I have not checked if the same happens in single player.

**Comment by MehVahdJukaar** [comment]
does this still happen?

**Comment by thekrugers** [comment]
I upgraded to the latest. For some reason ATLauncher did an upgrade of Supplementaries from 1.20-2.8.15 to 1.20-2.8.15 (same version number, but it saw it as an upgrade anyway). I also upgraded Moonlight to 1.20-2.12.1.
I did remember to do the upgrades on my server and client both.

Now I can't log in to my server, I get this error message when I try to log in to the server: Internal Exception: java.lang.IllegalStateException: Missing tag TagKey[minevcraft:item / supplementaries:hourglass_sands]
